external/netstandard/netstandard.depproj

```diff
@@ -23,6 +30,27 @@
       </Reference>
     </ItemGroup>
   </Target>
+
+  <Target Name="CopyToAdditionalOutputDirectories" 
```

Comment:
does it make sense to move this traget to one of the targets files? I am seeing this more generic Target.